src/components/Calculator/ClaculatorScreen.js

```diff
@@ -1,16 +1,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class CalculatorScreen extends React.Component {
-  render() {
-    const { value } = this.props;
-    return (
-      <div className="calculator-screen">
-        <p className="calculator-value">{value}</p>
-      </div>
-    );
-  }
+function CalculatorScreen({ value }) {
```

Comment:
- [ ] Good job so far by refactoring your code :clap:  But since React uses ES6, I think you should use some of the new features like Arrow Functions for the reason that it is a good practice and also this can make for example a lot easier to predict functions behavior when passed as callbacks, and prevents bugs caused by use of this within callbacks. :smile:  Please do it :+1: